packages/shared/src/hooks/useShareComment.ts

```diff
@@ -38,10 +40,12 @@ export function useShareComment(
       openShareComment: async (comment, post) => {
         if ('share' in navigator) {
           try {
+            const shortUrl = await getShortUrl(
+              `${post.commentsPermalink}${getCommentHash(comment.id)}`,
+              'share_comment',
```

Comment:
CID i guess, alternative comment, maybe we should make this object based? (just easier for everyone to understand what you are passing here)